src/igvfd/audit/measurement_set.py

```diff
@@ -11,9 +11,9 @@
 @audit_checker('MeasurementSet', frame='object')
 def audit_related_multiome_datasets(value, system):
     '''
-        audit_detail: Measurement sets with a specified multiome_size are expected to have the corresponding amount of links to other measurement sets (excluding itself) in related_multiome_datasets which are expected to have the same multiome_size and samples.
+        audit_detail: Measurement sets with a multiome size are expected to have the corresponding amount of links to other measurement sets (excluding itself) in related multiome datasets, which are expected to have the same multiome size and samples.
```

Comment:
Measurement sets with a multiome size are expected to have the corresponding amount of measurement sets (excluding itself) listed in related multiome datasets. Each of these datasets are expected to have the same multiome size and samples

Reply:
Ok.